app/src/main/res/layout-land/help_item.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data class="HelpItemBinding">
+
+    <variable
+      name="viewmodel"
+      type="org.oppia.app.help.HelpViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+      android:id="@+id/recycler_item_text_view"
```

Comment:
Please change this id to what this textview holds, meaning Ids must be contextual.

Reply:
Make sure that this is changed in test cases also, if it is getting used.

Reply:
Okay I will do this change once I reach home, am currently traveling.

Reply:
Done :)